projects/vdk-core/setup.cfg

```diff
@@ -52,7 +52,7 @@ install_requires =
     tenacity
 
 # Require a specific Python version
-python_requires = >=3.7, <3.11
+python_requires = >=3.7
```

Comment:
I wondered. 
If new python version comes before we "officially" support it people will not be able to install it even if everything works. Also looking at what other python distribution do
* requests set the next "major" version and upper bound
* boto, urlib3, certifi  do not set an upper bound at all. 

I will follow requests suite and set the next major version. I think it's expected that the major version would have breaking changes and a lot of things will not work ( you know how much broken between python 2 and 3). But minor versions should not

Reply:
> If new python version comes before we "officially" support it people will not be able to install it even if everything works. 

If a new version comes before we "officially" support it, will it be automatically tested? I think not, since an explicit change with the CICD is currently required, to do testing using an additional python version that the hardcoded ones.

So...the actual end users will be our primary tester, on wherever they had installed VDK. In case they hadn't pinned they environments to python 3.11, then their environments will become tester 0. Do we want that?

I would not recommend to shift the responsibility for first adoption of that which had not been yet tested before releasing, to the end users, making their deployment (or alternatively adding disclaimers they should pin, since we increase installation complexity). VDK should simply work if we installed it somewhere. It should be fast-fail at least.

Reply:
I agree with the new major version bound, makes sense to me